docs/js-core-1/week-1/lesson.md

```diff
@@ -239,6 +233,106 @@ Using online documentation, what other things can you do with the `Math` library
 
+
+You can also use expressions with string interpolation or as a `return` value.
+
+```js
+console.log(`2 + 4 is ${2 + 4}`); // 2 + 4 inside the string template evaluates to 6
+
+function double(num) {
+  return num * 2; // expression num * 2 being returned from this function
+}
+```
+
+## Statements and expressions
+
+In JavaScript, there are **expressions** and **statements**. 
+We will use these words frequently to describe code.
+### Statement
+
+A statement is some code that typically carries out an instruction. You have encountered statements already:
+
+```js
+const greeting = "Hello Code Your Future!";
+```
+
+The above piece of code is a **variable declaration**, which is also a statement. 
+This statement is just an instruction to store the value `"Hello Code Your Future!"` in the variable `greeting`.
+
+Statements end in a semi-colon, though they can be omitted in JavaScript.
+
+There are other different types of statements that we will learn about in the coming weeks.
+
+### Expression
+
+Any unit of code that can be evaluated to a value is known as an **expression**.<br />
+We say that an expression _evaluates to_ a value.
+We use expressions all the time in lot of different places:
+
+- they can be assigned to variables
+- returned from functions (more on this later)
+- passed as arguments to functions (more on this later)
+- combined with other expressions to form new expressions
+
+The following are all examples of expressions:
+
+```js
+1 + 1 // evaluates to 2
+"hello" // evaluates to "hello"
+2 * 4 // evaluates to 8
+"Hello" + "World" // evaluates to "HelloWorld"
+```
+
+We can take the value produced by an expression and assign it to a variable.
+
+```js
+let total = 1 + 1; // assign the expression 1 + 1 to a variable total
+let greeting = "hello"; // assign the expression "hello" to a variable greeting
+let product = 2 * 4; // assign the expression 2 * 4 to a variable product
+let worldGreeting = "Hello" + "World"; // assign the expression "Hello" + "World" to a variable worldGreeting
+```
+
+
+### Exercise ( 15 mins )
+
+You can run `node` by itself, which will open a _node console_, also called a [Read–Eval–Print Loop (REPL)](https://www.tutorialspoint.com/nodejs/nodejs_repl_terminal.htm).
+
+This console allows you to enter code in the console line by line and is a great way of testing bits of code before writing it in a script.
+
+In your terminal, run the command `node` and then enter a line of code from the code below and then run enter. Do this for each line of code in the list.
+Think about the following questions each time:
+
+What is the output in the terminal each time ?
+Is there anything you didn't expect ?
+
+Can you work out which of the lines of code below are **expressions** and which are not ?
+
+_Remember it can only be an expression if you can place it on the right hand side of an assignment._
```

Comment:
This feels a bit out of place, since it's the first time it's come up.

What do you think about adding a (small) section comparing expressions and statements, where we could show a code snippet with the differences?